src/itempicker.cpp

```diff
@@ -19,9 +19,20 @@ bool ItemPicker::eventFilter(QObject* obj, QEvent* event) {
 
   m_lastEvent = event;
 
-  if (event->type() != QEvent::MouseButtonPress &&
-      event->type() != QEvent::MouseButtonDblClick &&
-      event->type() != QEvent::MouseButtonRelease) {
+  bool isMouseEvent = event->type() == QEvent::MouseButtonPress ||
+                      event->type() == QEvent::MouseButtonDblClick ||
+                      event->type() == QEvent::MouseButtonRelease;
+  bool isTouchEvent =
+      event->type() == QEvent::TouchBegin &&
+      !!qobject_cast<QQuickWindow*>(static_cast<QTouchEvent*>(event)->target());
+
+  if (
+#if defined(MVPN_ANDROID) || defined(MVPN_IOS)
+      !isTouchEvent
```

Comment:
We need to test this with a desktop build with a touch screen.

Reply:
good point